CHANGELOG.md

```diff
@@ -12,6 +12,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Removed
 
 ### Changed
+- Decoupled EventController from AudioVideo and MeetingSession
```

Comment:
I'm assuming we are going to have an in depth transition guide so its not a big deal to not go into details here.

Reply:
Yes, although that actually is supposed to be part of this commit, I've gone ahead and added it as `guides/17_Migration_to_3_0.md`